src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/AbstractKubernetesPipelineTest.java

```diff
@@ -72,7 +72,7 @@
     @Rule
     public JenkinsRuleNonLocalhost r = new JenkinsRuleNonLocalhost();
     @Rule
-    public LoggerRule logs = new LoggerRule().recordPackage(KubernetesCloud.class, Level.ALL);
+    public LoggerRule logs = new LoggerRule().recordPackage(KubernetesCloud.class, Level.FINE);
```

Comment:
Yeah I considered this during #708. For some tests it is actually important to follow the details of pod calculation and so on. But, we can readd finer-level logging on particular classes to particular tests as we find it useful.